dockers/docker-fpm-frr/frr/frr.conf.j2

```diff
@@ -13,9 +13,27 @@ agentx
 !
 {% include "zebra/zebra.interfaces.conf.j2" %}
 !
+{% if DEVICE_METADATA.localhost.frr_mgmt_framework_config is defined and DEVICE_METADATA.localhost.frr_mgmt_framework_config == "true" %}
+{% if MGMT_VRF_CONFIG %}
+{% if MGMT_VRF_CONFIG['vrf_global']['mgmtVrfEnabled'] == 'false' %}
+{% include "staticd.default_route.conf.j2" %}
+{% endif %}
+{% else %}
+{% include "staticd.default_route.conf.j2" %}
+{% endif %}
+!
+{% include "staticd.db.conf.j2" %}
```

Comment:
nit: same a comment line preceding the rendered template would be great here.

Reply:
Addressed